package.json

```diff
@@ -11,6 +11,12 @@
         "url": "https://github.com/sponsors/ljharb"
     },
     "main": "lib/index.js",
+    "exports": {
```

Comment:
adding "exports" is a breaking change. this should also include the ability to access package.json.

Reply:
People actually access other people's package.json and assume it's a part of the public API of packages? Never heard of that

Reply:
Everything reachable is public, yes - and the lack of package.json access broke preact users recently.

Reply:
```suggestion
    "exports": [{](url)
```